.github/workflows/test-build-publish.yml

```diff
@@ -10,56 +11,56 @@ on:
       - synchronize
 
 jobs:
-    test:
-        name: Check code against linter/unit tests
-        runs-on: ubuntu-18.04
-        strategy:
-          matrix:
-            python-version: [3.6, 3.7, 3.8, 3.9]
-        steps:
-        - uses: actions/checkout@master
-        - name: Set up Python ${{ matrix.python-version }}
-          uses: actions/setup-python@v2
-          with:
-            python-version: ${{ matrix.python-version }}
-        - name: Install dependencies
-          run: python -m pip install tox tox-gh-actions
-        - name: Test with Tox
-          run: tox
+  test:
+    name: Check code against linter/unit tests
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        python-version: [3.7, 3.8, 3.9]
```

Comment:
Do we need to make some updates for adding `3.10` and `3.11` python versions?

Reply:
Would be nice to have.